spec/requests/v6/assessments_controller/simple_spec.rb

```diff
@@ -2137,6 +2137,48 @@ def cash_transactions(amount)
           end
         end
       end
+
+      context "MTR" do
+        let(:current_date) { Date.new(2525, 0o5, 0o1) }
+
+        context "when threshold 'fixed_employment_allowance' is set to 60" do
```

Comment:
this isn't even true (its 66) but the context should describe the 'given' which is MTR?

Reply:
Removed request spec and added unit test instead